components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -2886,35 +2890,15 @@ def listColleagues(self):
         """
 
         default = self.getObject(
-            "ExperimenterGroup", self.getEventContext().groupId)
+            "ExperimenterGroup", self.getEventContext().groupId,
+            opts={'load_experimenters': True})
         if not default.isPrivate() or self.isLeader():
             for d in default.copyGroupExperimenterMap():
                 if d is None:
                     continue
                 if d.child.id.val != self.getUserId():
                     yield ExperimenterWrapper(self, d.child)
 
-    def groupSummary(self, gid=None, exclude_self=False):
```

Comment:
Seems that webtagging is still using this.
So I will leave it until webtagging has updated.